apps/condo/lang/ru.json

```diff
@@ -223,6 +223,7 @@
   "pages.condo.property.warning.modal.Title": "Не сохранять дом?",
   "pages.condo.warning.modal.SaveLabel": "Сохранить",
   "pages.condo.warning.modal.LeaveLabel": "Не сохранять",
+  "pages.condo.warning.modal.HelpMessage": "",
```

Comment:
Where this key is used?
If nowhere, why it was added?

Reply:
It is not used and was added on task where we show modal window preventing user from leaving form without saving. I searched for all code and it is not used